index.html

```diff
@@ -47,5 +48,86 @@ <h2>I’am a Software Developer</h2>
         </div>
     </section>
     </div>
+    <section class="grid-work">
+      <div class="work-heading"><h1>My Recent Works</h1>
+      <hr/>
+      </div>
+      <div class="photo-work">
+        <img src="./Pictures/ImgPlaceholder.png" alt="Work Photo">
+         
+        <div class="photo-text">
+          <h1>Multi-Post Stories</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+     
+      <div class="work-list">
+        <ul>
+          <a href="#"><li class="item work-item-1">css</li></a>
+          <a href="#"><li class="item work-item-2">html</li></a>
+          <a href="#"><li class="item work-item-3">bootstrap</li></a>
+          <a href="#"><li class="item work-item-4">Ruby</li></a>
+        </ul>
+      </div>
+      <div class="work-link">
+        <a href="#">See Project</a>
+      </div>
+      </div>
+      <div class="project">
+        <div class="project-heading">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      </div>
+      <div class="project-lists">
+        <ul>
+          <a href="#"><li class="project-item-1">html</li></a>
+          <a href="#"><li class="project-item-2">bootstrap</li></a>
+          <a href="#"><li class="project-item-3">Ruby</li></a>
+        </ul>
+      </div>
+      <div class="project-list">
+        <a href="#">See Project</a>
+      </div>
+      </div>

```

Comment:
- [ ] `required`
- [ ] please consider a space between the 2 cards the Figma template
- [ ] also you may consider making the button stretch all the way as the Figma template
- [ ] please see screenshots belows

`your template`
![image](https://user-images.githubusercontent.com/66230306/198733115-7c5cb08a-833a-4af8-8ef9-69ed297201d0.png)

`figma tempate`
![image](https://user-images.githubusercontent.com/66230306/198734571-05c33656-a893-4fe6-8d47-c1093f7dc87b.png)